Dockerfile

```diff
@@ -1,38 +1,28 @@
-FROM ubuntu:14.04.4
+FROM ubuntu:14.04
 
-# Replace shell with bash so we can source files
-RUN rm /bin/sh && ln -s /bin/bash /bin/sh
-
-# Install apt package dependencies
-RUN apt-get clean all && apt-get update && \
-    apt-get -y install wget curl python make g++ git supervisor && \
```

Comment:
We need supervisor as the process that keeps the container alive always. Typical docker containers are supposed to be single process, but we had to take a very hybrid approach for our stakeholder's needs.